cli/lib/exec/open.js

```diff
@@ -48,11 +49,21 @@ module.exports = {
   start (options = {}) {
     const args = processOpenOptions(options)
 
+    function getOpenModeEnv () {
+      if (os.platform() !== 'linux') return
+
+      // on non-GTK desktop environments (KDE, XFCE, ...), native file pickers will not be used in Electron unless
+      // we explicitly opt-in the the GTK portal functionality
+      // @see https://tristan.partin.io/blog/2021/04/01/electron-linux-and-your-file-chooser/
+      return { GTK_USE_PORTAL: '1' }
```

Comment:
Just curious, is this something that could be validated and set in `packages/electron` instead of needing to setup & pass this ENV around?